docs/coding-guidelines.md

```diff
@@ -48,7 +48,7 @@ The general rule we follow is "use Visual Studio defaults".
 
 1. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member` rather than `_member`), the existing style in that file takes precedence. Changes/refactorings are possible, but depending on the complexity, change frequency of the file, might need to be considered on their own merits in a separate pull request.
 
-1. We only use `var` when it's obvious what the variable type is.
+1. Don't use `var` when it's not obvious what the variable type is.
```

Comment:
Rephrasing to avoid the double use of "not" and phrasing as more what we should do.

```suggestion
1. Explicitly state the data type, and avoid using `var`, in variable declarations where the type is not obvious.
```